models/stg_reddit_ads.yml

```diff
@@ -332,7 +332,7 @@ models:
       - name: account_id
         description: '{{ doc("account_id") }}'
       - name: post_id
-        description: '{{ doc("post_id") }}'
+        description: '{{ doc("post_id") }}' 
```

Comment:
Should we also provide a not_null test for this field?

Reply:
updated